Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -351,12 +350,11 @@ theorem det_pos [DecidableEq n] {M : Matrix n n ℝ} (hM : M.PosDef) : 0 < det M
   intro i _
   rw [hM.isHermitian.eigenvalues_eq]
```

Comment:
I think this works to close the goal here.
```suggestion
  exact hM.isHermitian.eigenvalues_pos i
```

Reply:
Almost, but without the `isHermitian`.